### DISCUSSION TITLE
Shared WorkBench Data Sets

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5708] [Feature Requests]
**Requested By:** Heather Cole at AAFC-AAC (DAO)

> 
We are just getting into S7, and really excited for new Workbench options. I see lots of great info about how to use the Workbench, but I couldn’t find anything clear about whether multiple users can have access to the same workbench dataset. For example, I create a dataset in the workbench, partially populate it, then want someone else (or multiple other people to have view/edit access to it.

**Comment by grantfitzsimmons** [comment]
We could engineer a solution where there is a new field added. Maybe something like `isLocked` or similar. This could be set to true when someone is editing a data set and unlocked once they exit the session properly. 

There could be the option to override this lock if the user has permissions.

**Comment by maxpatiiuk** [comment]
As a short-term workaround, does data set transfer work for them?

**Comment by grantfitzsimmons** [comment]
Maybe– I sent her brief instructions on how to use that feature 
https://discourse.specifysoftware.org/t/multiple-user-access-and-permissions-to-workbench-datasets/976/2?u=specify
